subprojects/docs/src/snippets/antMigration/fileDeps/groovy/build.gradle

```
@@ -12,7 +12,7 @@ repositories {
 
 dependencies {
     implementation files('libs/our-custom.jar')  // <2>
-    implementation ':log4j:1.2.8', ':commons-io:2.1'  // <3>
+    implementation ':log4j-core:2.19.0', ':commons-io:2.11.0'  // <3>
```

:x: still uses the old jar names
```suggestion
    implementation ':awesome-framework:2.0'  // <3>
    implementation ':utility-library:1.0'  // <3>
```
BTW I moved this to two lines so it is aligned with the Kotlin snippet.